cluster/v1alpha1/types.go

```diff
@@ -211,7 +211,7 @@ type ResourceModel struct {
 // ResourceModelRange describes the detail of each modeling quota that ranges from min to max.
 // Please pay attention, by default, the value of min can be inclusive, and the value of max cannot be inclusive.
 // E.g. in an interval, min = 2, max =10 is set, which means the interval [2,10).
-// This rule ensure that all intervals have the same meaning. If the last interval is +8,
+// This rule ensure that all intervals have the same meaning. If the last interval is +∞,
```

Comment:
I missed it, still problem.

```
cluster/v1alpha1/types.go
cluster/types.go
```

Reply:
Seems no problem here. 
But I think we can replace the `+∞` with `infinite`. 

By the way, would like to send a PR to karmada repo for the replacement?